pyvista/core/dataobject.py

```diff
@@ -252,17 +252,7 @@ def _repr_html_(self):  # pragma: no cover
         raise NotImplementedError('Called only by the inherited class')
 
     def copy_meta_from(self, *args, **kwargs):  # pragma: no cover
-        """Copy pyvista meta data onto this object from another object.
-
-        Parameters
-        ----------
-        ido : pyvista.DataObject
-            Data object to copy the metadata from.
-
-        deep : bool, optional
-            Deep or shallow copy.
-
-        """
+        """Copy pyvista meta data onto this object from another object. Intended to be overridden by subclasses."""
         pass  # called only by the inherited class
```

Comment:
Shouldn't this either `raise NotImplementedError()` or be an abstract method? Or is it OK if this method is a silent no-op?

Reply:
It is already an abstract class, right ?

Reply:
Silent no op is fine for this. Especially since the new Cell class inherits it but doesn't use it. Leaving it no op prevents the need the for a no op implementation in the Cell class (and maybe other classes that don't implement? DataSet might be the only class that uses it? On mobile so not sure)